aws_google_auth/prepare.py

```diff
@@ -17,7 +17,8 @@ def get_prepared_config(
     def default_if_none(value, default):
         return value if value is not None else default
 
-    google_config.profile = default_if_none(profile, google_config.profile)
+    # If no profile is specified, default to "sts" so we don't clobber the user's default.
+    google_config.profile = default_if_none(profile, google_config.profile) or "sts"
```

Comment:
Oh shoot; I had a lot of PR's going on and it looks like I may have branched off a branch instead of master. I'll put a patch to reduce the changes back down.

Reply:
Doing it here (instead of `__init__.py:33`will enable us to use the tests you previously created (because `prepare.py` gets called from those tests). What are your thoughts on the tests?